examples/chip-tool/commands/icd/ICDCommand.cpp

```diff
@@ -43,7 +43,7 @@ CHIP_ERROR ICDListCommand::RunCommand()
                 info.start_icd_counter, info.offset, ChipLogValueX64(info.monitored_subject));
 
         static_assert(std::is_same<decltype(CHIPCommand::sSessionKeystore), Crypto::RawKeySessionKeystore>::value,
-                      "DefaultSessionKeystore` is expected to be `RawKeySessionKeystore`");
+                      "`CHIPCommand::sSessionKeystore` is expected to be `RawKeySessionKeystore`");
```

Comment:
Message should explain _why_ it's expected to be that, no?  The expectation is clear from the assert condition.

Reply:
resolved, thanks